### ISSUE TITLE
Can not import psutil on pypy3 (travis CI)

### ISSUE THREAD

**Issue by pwilczynskiclearcode** [giampaolo/psutil#853]
``` python
 import psutil
../../../virtualenv/pypy3-2.4.0/site-packages/psutil-4.2.0-py3.2-linux-x86_64.egg/psutil/__init__.py:78: in <module>
    from . import _pslinux as _psplatform
../../../virtualenv/pypy3-2.4.0/site-packages/psutil-4.2.0-py3.2-linux-x86_64.egg/psutil/_pslinux.py:24: in <module>
    from . import _psutil_linux as cext
../../../virtualenv/pypy3-2.4.0/site-packages/psutil-4.2.0-py3.2-linux-x86_64.egg/psutil/_psutil_linux.py:7: in <module>
    __bootstrap__()
../../../virtualenv/pypy3-2.4.0/site-packages/psutil-4.2.0-py3.2-linux-x86_64.egg/psutil/_psutil_linux.py:6: in __bootstrap__
    imp.load_dynamic(__name__,__file__)
E   ImportError: unable to load extension module '/home/travis/virtualenv/pypy3-2.4.0/site-packages/psutil-4.2.0-py3.2-linux-x86_64.egg/psutil/_psutil_linux.pypy3-24.so': /home/travis/virtualenv/pypy3-2.4.0/site-packages/psutil-4.2.0-py3.2-linux-x86_64.egg/psutil/_psutil_linux.pypy3-24.so: undefined symbol: PyModule_GetState
```

Tested both psutil `4.2.0`, `4.3.0`, both causing the same exception.

**Comment by giampaolo** [comment]
If you manage to address this issue please let me know. It would be good to add instructions on how to build/install psutil on pypy in https://github.com/giampaolo/psutil/blob/master/INSTALL.rst.

**Comment by pwilczynskiclearcode** [comment]
There's no `pypy3-dev` package and it works correctly on `pypy2`. I found a similar issue: https://github.com/giampaolo/pysendfile/issues/28

**Comment by giampaolo** [comment]
LOL that's also another lib of mine. By googling for "pypy PyModule_GetState" it seems this is a problem with PYPY not supporting `PyModule_GetState` so unless there's a substitute for it in PYPY I suppose we can't do anything about it.

**Comment by ryanli1994** [comment]
Guys any plan to fix this ? 😭

**Comment by giampaolo** [comment]
Took a look at this and I can reproduce the issue. Unfortunately I don't know how to fix it. It would seem pypy3 is not compatible with some C Python APIs. Looks like you'd have more luck asking on pypy mailing list / bugtracker / whatever.

**Comment by pirate** [comment]
For reference, the issue has been [reported via the pypy issue tracker](https://bitbucket.org/pypy/pypy/issues/2450/cant-import-psutil-on-pypy3-missing).

**Comment by giampaolo** [comment]
Furthermore, AFAIK in the Python user land PyModule_GetState is used by all C extension modules, so this would really mean pyp3 does not support any of them, not only psutil.

**Comment by giampaolo** [comment]
https://mail.python.org/pipermail/pypy-dev/2017-May/015135.html

**Comment by giampaolo** [comment]
This was fixed in latest pypy night build (pypy3 5.8.0):
https://bitbucket.org/pypy/pypy/issues/2450/cant-import-psutil-on-pypy3-missing
